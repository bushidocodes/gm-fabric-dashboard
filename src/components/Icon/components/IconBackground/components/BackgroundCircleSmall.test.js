import React from "react";
import { shallow } from "enzyme";
import BackgroundCircleSmall from "./BackgroundCircleSmall";

describe("BackgroundCircleSmall", () => {
  it("matches snapshot", () => {
    const aBackgroundCircleSmall = shallow(<BackgroundCircleSmall />);
    expect(aBackgroundCircleSmall).toMatchSnapshot();
  });
});
