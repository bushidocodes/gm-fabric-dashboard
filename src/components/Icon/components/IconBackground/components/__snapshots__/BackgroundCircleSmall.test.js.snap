// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundCircleSmall matches snapshot 1`] = `
<g
  className="iconBackground"
  fill="currentColor"
  fillOpacity="1"
  fillRule="evenodd"
  id="Square"
>
  <path
    d="M12,21 C7.02943725,21 3,16.9705627 3,12 C3,7.02943725 7.02943725,3 12,3 C16.9705627,3 21,7.02943725 21,12 C21,16.9705627 16.9705627,21 12,21 Z"
    id="BackgroundCircleSmall"
  />
</g>
`;
